generative_ai/embeddings/requirements.txt

```
@@ -7,8 +7,8 @@ google-cloud-aiplatform[all]==1.69.0
 sentencepiece==0.2.0
 google-auth==2.29.0
 anthropic[vertex]==0.28.0
-langchain-core==0.2.11
-langchain-google-vertexai==1.0.6
+langchain-core==0.2.33
+langchain-google-vertexai==1.0.10
```

Sample failing for unrelated reasons right now, remove update
```suggestion
langchain-core==0.2.11
langchain-google-vertexai==1.0.6
```